core/node/api_server/src/web3/namespaces/zks.rs

```diff
@@ -352,14 +352,14 @@ impl ZksNamespace {
         let (root, mut proof) = MiniMerkleTree::new(merkle_tree_leaves, Some(tree_size))
             .merkle_root_and_path(l1_log_index);
 
+        // FIXME Definitely refactor all of it
```

Comment:
seems we have a bunch of 'helper' printf here...

Can you create a linear ticket and add it here (so that we don't forget).

Reply:
I don't want to create a ticket because I'm not yet sure what to put there. It's on my radar, I will get to it soon